Samples/outlook-set-signature/manifest.json

```diff
@@ -0,0 +1,263 @@
+{
+    "$schema": "http://zlatkom2:32080/metaos/schema/metaos.public.schema.json#",
```

Comment:
This schema needs to be updated. So is the "file" field below. It should be changed to "url".

Reply:
The schema reference needs to be updated to the final public location once we have that.  As far as I know that schema will use "file" and not "url" so that doesn't need to be changed.